tests/baselines/reference/customTransforms/sourceMapExternalSourceFiles.js

```diff
@@ -0,0 +1,7 @@
+// [source.js.map]
+{"version":3,"file":"source.js","sourceRoot":"","sources":["source.ts","another.html"],"names":[],"mappings":"AAAA,iCACyB,CAAA;ACDzB,QAAS,CDED"}
+
+// [source.js]
+"multi\n                    line";
+'change';
```

Comment:
Similarly here?

Reply:
Since we are changing only sourcemap range and not text it looks like changed.